packages/react-instantsearch-hooks-dom/src/ui/HierarchicalMenu.tsx

```diff
@@ -7,35 +7,35 @@ import type { useHierarchicalMenu } from 'react-instantsearch-hooks';
 
 type HierarchicalMenuClassNames = {
   /**
-   * Class names to apply to the root element.
+   * Class names to apply to the root element
    */
   root: string;
   /**
-   * Class names to apply to the root element, when there's no refinement possible.
+   * Class names to apply to the root element when there are no refinements possible
    */
   rootNoRefinement: string;
   /**
-   * Class names to apply to the list element.
+   * Class names to apply to the list element
    */
   list: string;
   /**
-   * Class names to apply to any item element.
+   * Class names to apply to each item element
    */
   item: string;
   /**
-   * Class names to apply to a selected item element.
+   * Class names to apply to the selected item
    */
-  selectedItem: string;
+  itemSelected: string;
```

Comment:
Why this change? It's `selectedItem` in InstantSearch.js.

Reply:
I checked the other Hooks DOM components and made the change for consistency: https://github.com/algolia/react-instantsearch/search?q=itemSelected.

Reply:
We should aim for consistency with IS.js imo.

Reply:
You're right. I created a [task](https://algolia.atlassian.net/browse/FX-1324) to list all the adjustments we should make on the components before releasing them.